### PR TITLE
Fixes #14638: Add a generic method to add ace for "other" on a given file

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/14638

### CODE REVIEW

**Comment by Fdall** [comment]
PR rebased

File: tree/30_generic_methods/permissions_other_acl_present.cf [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+# user:bob:rwx
+# group::r--
+# mask::rwx
+# other::--x
+#
+# ~~~~
+#
+# @parameter path       Path of the file or directory
+# @parameter recursive  Recursive Should ACLs cleanup be recursive, "true" or "false" (defaults to "false")
+# @parameter other_ace  ACE to enforce for the given other.
```
Comment (ncharles) [inline tree/30_generic_methods/permissions_other_acl_present.cf]: why the given other ? simply for "other"

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

File: tree/30_generic_methods/permissions_other_acl_present.cf [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+      "sanitize single reports" usebundle => _classes_sanitize("${inner_class_prefix[${file_list}]}");
+
+    pass3.files_found::
+      "merge reports"    usebundle => _classes_combine_two("${class_prefix}", "${inner_class_prefix[${file_list}]}", "${class_prefix}");
+      "sanitize reports" usebundle => _classes_sanitize("${class_prefix}");
+      "set old classes"  usebundle => _classes_copy("${class_prefix}", "${old_class_prefix}");
+
+    is_recursive.pass3.files_found::
+      "report" usebundle => _log_v3("${report_string} ${const.endl}Set POSIX acl ${other} recursively on ${path}", "${path}", "${old_class_prefix}", "${class_prefix}", @{args});
```
Comment (ncharles) [inline tree/30_generic_methods/permissions_other_acl_present.cf]: on the rudder cli, it output "repaired: /tmp/acl", but in the interface it's ok

**Pull request merged**